docs/root/intro/arch_overview/upstream/load_reporting_service.rst

```diff
@@ -12,4 +12,4 @@ to a node it is interested in getting the load reports for. Envoy in this node w
 :ref:`LoadStatsRequest <envoy_v3_api_msg_service.load_stats.v3.LoadStatsRequest>`. This is done periodically
 based on the :ref:`load reporting interval <envoy_v3_api_field_service.load_stats.v3.LoadStatsResponse.load_reporting_interval>`
 
-Envoy config with LRS can be found at :repo:`/examples/load-reporting-service/service-envoy-w-lrs.yaml`.
+Envoy config with LRS can be found at :repo:`envoy.yaml </_include/load-reporting-service/envoy.yaml>`.
```

Comment:
```suggestion
Envoy config with LRS can be found at `envoy.yaml </_include/load-reporting-service/envoy.yaml>`.
```

no `repo:` at beginning as this is reffing an rst artefact not linking to repo

Reply:
ok, I'll remove the :repo:, but I think that this didn't work previously.